# Statistical Analysis of High-Throughput Genomic and Transcriptomic Data 
Fall/Herbst-semester 2020

## Lectures
Mondays 9.00-9.45 (remote), 10.00-10.45 (remote)

## Exercises
Monday 11.00-11.45 (remote)

## Lecturers

[Dr. Hubert Rehrauer](http://www.fgcz.ch/the-center/people/rehrauer.html), Group Leader of Genome Informatics at [FGCZ](http://www.fgcz.ch/)  

[Prof. Dr. Mark Robinson](https://robinsonlabuzh.github.io/), Associate Professor of Statistical Genomics, [IMLS](http://www.imls.uzh.ch/index.html), UZH  

## Teaching Assistant

[Ahmadreza Yousefkhani](https://ir.linkedin.com/in/ahmadreza-yousefkhani-3b2a99a8), M.Sc. student in Computational Biology and Bioinformatics  


## Schedule

| Date  | Lecturer | Topic | Exercise | JC1 | JC2 |
| --- | --- | --- | --- | --- | --- |
| 14.09.2020  | Mark + Hubert  | admin; mol. bio. basics | R markdown; git(hub) | <img width=100/> | <img width=100/> |
| 21.09.2020  | Mark | interactive technology/statistics session  | group exercise: technology pull request | | |
| 28.09.2020  | Hubert | NGS intro; exploratory data analysis | EDA in R | | |
| 05.10.2020  | Hubert | mapping  | Rsubread | | |
| 12.10.2020  | Mark | limma + friends | linear model simulation + design matrices | | |
| 19.10.2020  | Hubert | RNA-seq quantification    | RSEM  | | |
| 26.10.2020  | Mark | edgeR+friends 1 | basic edgeR/voom | |  |
| 02.11.2020  | Mark | edgeR+friends 2  | GLM/DEXSeq | | |
| 09.11.2020  | Kathi | hands-on session #1: RNA-seq  | FASTQC/Salmon/etc. | [Statistical significance for genomewide studies](https://www.pnas.org/content/100/16/9440) (RA, HH) | [Identifying signaling genes in spatial single cell expression data](https://www.biorxiv.org/content/10.1101/2020.07.27.221465v2) (YM, KD, GJ) |
| 16.11.2020  | Hubert | single-cell 1: preprocessing, dim. reduction, clustering | | [Testing hypotheses about the microbiome using the linear decomposition model](https://www.biorxiv.org/content/10.1101/229831v3.full.pdf+html) (FS,BO,SA) | [Generalizing RNA velocity to transient cell states through dynamical modeling](https://www.nature.com/articles/s41587-020-0591-3?proof=t) (DP, EH) |
| 23.11.2020  | Helena | hands-on session #2: cytometry  | cytof null comparison | (Richard Affolter, Philip Hartout, Martin Emons) paper: tbd | [Empirical Bayes Analysis of a Microarray Experiment](https://www.tandfonline.com/doi/abs/10.1198/016214501753382129) (Jennifer Probst, Eljas RÃ¶llin, Lisa Herzog) |
| 30.11.2020  | Mark | single-cell 2: cell type definition, differential state  | scRNA exercise 2 | [A Bayesian mixture model for the analysis of allelic expression in single cells](https://doi.org/10.1038/s41467-019-13099-0) (Sneha-Sundar,SmaragdaDimitrakopoulou,marinapanteli) | [scMET: Bayesian modelling of DNA methylation heterogeneity at single-cell resolution](https://www.biorxiv.org/content/10.1101/2020.07.10.196816v2) (MW, RM, DW) |
| 07.12.2020  | Pierre-Luc | hands-on session #3: single-cell RNA-seq  | full scRNA-seq pipeline | [ScreenBEAM: a novel meta-analysis algorithm for functional genomics screens via Bayesian hierarchical   modeling](https://academic.oup.com/bioinformatics/article/32/2/260/1743949) (Leonor Schubert, Jonathan   Haab, Flavio Rump) | [Detection of differentially abundant cell subpopulations discriminates biological states in scRNA-seq data] (https://www.biorxiv.org/content/10.1101/711929v3) (TE, RB, AB) 
| 14.12.2020  | Mark | loose ends: HMM, EM, robustness   | segmentation, peak finding | [NEBULA: a fast negative binomial mixed model for differential expression and co-expression analyses of large-scale multi-subject single-cell data](https://www.biorxiv.org/content/10.1101/2020.09.24.311662v1) (HML, SCD, SW) | X |



## Course material

Assuming you have git installed locally, you can check out the entire set of course materials with the following command (from command line):
```
git clone https://github.com/sta426hs2020/material.git
```  
Alternatively, for a ZIP file of the repository, you can click on the (green) 'Clone or download' (top right) and then click 'Download ZIP'.
